### PR TITLE
Updated qos-interative information and other changes

### PR DESCRIPTION
**Pull request by jeremysmith123**
Reworked introduction to running jobs on slurm under tech docs.
Changed information about --qos qos-interative and its use.
Added information about change a user's default slurm account.
Removed Nicky's contact email from contacts.

### CODE REVIEW

File: docs/getting_started/submit_job_slurm.md [hunk 1]
```diff
@@ -36,55 +36,57 @@ In the shell script above, the application you wish to run during the job is des
...
 
-```shell
+```bash
 	$ sbatch my_slurm_script.sh
 ```
 
 This will submit the job to the SLURM queue. If the requested resources are available the job will be initiated. You can see the status of all jobs in the SLURM queue by using the command:
 
-```shell
+```bash
```
Comment (kennedydane) [inline docs/getting_started/submit_job_slurm.md:45]: Think these are effectively synonymous?

File: docs/tech_docs/running_jobs.md [hunk 2]
```diff
@@ -232,7 +232,7 @@ The following table lists the parameters that can be used to describe the requir
...
 5. nodes refers to a single compute node or SLURM worker, i.e. one node that has 32 CPUs and 232 GB RAM
 6. The partition is the specific part of the cluster your job will run. You will only set this if you are [running a GPU job](/getting_started/submit_job_slurm?id=notes-for-gpu-jobs).
-7. To find your default account you can run the command `sacctmgr show User -p | grep ${USER}`, while the command `sacctmgr show Associations User=${USER} -p | cut -f 2 --d="|"` will show all your valid accounts. Note that it is only important that you set the account parameter if you are associated with more than one project on the cluster.
+7. To find your default account you can run the command `sacctmgr show User -p | grep ${USER}`, while the command `sacctmgr show Associations User=${USER} -p | cut -f 2 --d="|"` will show all your valid accounts. Note that it is only important that you set the account parameter if you are associated with more than one project on the cluster. You can change your default account using `sacctmgr modify user name=${USER} set DefaultAccount=<account>`, where `<account>` is one of your valid accounts.
```
Comment (kennedydane) [inline docs/tech_docs/running_jobs.md:235]: Have you confirmed regular users can do this?

File: docs/getting_started/submit_job_slurm.md (hunk 1 above)
Comment (jeremysmith123) [reply docs/getting_started/submit_job_slurm.md:45]: Yeah, and the formatting doesn't change on the website, I just wanted the text to be consistent.

File: docs/tech_docs/running_jobs.md (hunk 2 above)
Comment (jeremysmith123) [reply docs/tech_docs/running_jobs.md:235]: Yes I've confirm a user can change they own default account. I am not able to change the default of other users without sudo.

Comment (Jordatious) [reply docs/tech_docs/running_jobs.md:235]: Cool, I didn't realise there's a `${USER}` environment variable! I usually use `$(whoami)`!

File: docs/tech_docs/running_jobs.md [hunk 3]
```diff
@@ -2,11 +2,9 @@
 
 There are several methods to run jobs on Ilifu.
 
-1. Users may get access to a Jupyter Lab spawner mechanism on the cluster. This provides a Jupyter lab environment where users select what size compute node they need to run. This is a good environment for users to experiment with code and visualise certain results. This environment will be made available at [jupyter.ilifu.ac.za](https://jupyter.ilifu.ac.za)
+1. Users may access a JupyterLab environment on the cluster. This provides a JupyterLab environment where users select what size compute node they need to run. This is a good environment for users to experiment with code and visualise certain results. This environment is available at [jupyter.ilifu.ac.za](https://jupyter.ilifu.ac.za)
 
-2. Users can log into the SLURM headnode at and submit tasks to the SLURM queue. This environment will be made available at [slurm.ilifu.ac.za](https://slurm.ilifu.ac.za).
-
-3. Selected users will be given access to the OpenStack dashboard in order to have full control over the setup of a computing environment. This is available at [dashboard.ilifu.ac.za](https://dashboard.ilifu.ac.za)
```
Comment (Jordatious) [inline docs/tech_docs/running_jobs.md:9]: Why don't we want this anymore?

Comment (Jordatious) [reply docs/tech_docs/running_jobs.md:9]: Point 3 that is

**Comment by Jordatious** [comment]
Looks good to me. Happy to merge.

File: docs/tech_docs/running_jobs.md (hunk 3 above)
Comment (jeremysmith123) [reply docs/tech_docs/running_jobs.md:9]: It's not really for 'running jobs'. I think we should provide better information about bespoke services through Openstack. The entry that I want to remove I think just causes more confusion than it helps. Also selected users is poor wording.

Comment (Jordatious) [reply docs/tech_docs/running_jobs.md:9]: Ok sounds good. Let's try to capture this elsewhere.

**Pull request merged**